prow/cmd/deck/static/pr.html

```diff
@@ -41,6 +41,27 @@
     <main id="main-container" class="mdl-layout__content">
     </main>
 </div>
+<dialog id="search-dialog" class="pr-status-dialog mdl-dialog">
+    <h4 class="mdl-dialog__title">Github search query</h4>
+    <div class="mdl-dialog__content">
+        <p>You can use the Github search query to search for <strong>pull requests</strong> using Github search syntax.</p>
```

Comment:
nit: "You can query for pull request using Github search syntax." is a little more concise.